l10n/en/cms.ftl

```diff
@@ -0,0 +1 @@
+block-step-card-step = Step
```

Comment:
@stevejalim I'm not sure if this is the best place for the FTL strings for the CMS blocks, but my thinking is that we'll only have a handful of those since almost none of the strings in the CMS will be hardcoded. I thought of prefixing them with `block-BLOCK_NAME-TOKEN`. What do you think?